js/data.js

```diff
@@ -849,6 +1003,17 @@ var data = {
         dozent: ["Dr. Matthias Uflacker", "Dipl.-Kfm. Jens Krüger", "Thomas Kowark", "Franziska Häger"],
         nameLV: "Global Team-Based Product Innovation & Engineering III - ME 310 Coaching Research"
     },
+    programmierungVerteilterSysteme: {
+        kurz: "ParProg",
```

Comment:
There is no other short title "ParProg". There could be for Konzepte Paralleler Programmierung, but the short title wasn't named that way.